components/org.wso2.carbon.identity.application.authenticator.oidc/src/test/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticatorTest.java

```diff
@@ -201,8 +201,13 @@ public void init() {
         return new String[][]{
                 // When all parameters not null.
                 {"openid", "active,OIDC", "BASIC", "Error Login.", "true", "active", "Invalid can handle response for the request.", "Invalid context identifier."},
-                // When gran_type and login_type are null.
-                {null, "active,OIDC", null, "Error Login.", "true", "active", "Invalid can handle response for the request.", "Invalid context identifier."},
+                // When grant_type is null.
+                {null, "active,OIDC", "BASIC", "Error Login.", "false", "active", "Invalid can handle response for the request.", "Invalid context identifier."},
```

Comment:
Seems like the characters per line is > 120. If so please format it.

Reply:
Resolved with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/83/commits/f526303fadccc8122aaf11c8edb5b65a286745e7.